tests/search_disk_index.cpp

```diff
@@ -327,6 +352,15 @@ int main(int argc, char** argv) {
                        po::bool_switch()->default_value(false),
                        "Include full precision data in the index. Use only in "
                        "conjuction with compressed data on SSD.");
+    desc.add_options()(
+        "num_starting_points",
+        po::value<uint32_t>(&num_starting_points)->default_value(0),
+        "Number of starting points used for building index (defaults to 0");
+    desc.add_options()(
+        "selection_strategy_of_starting_points",
+        po::value<std::string>(&selection_strategy_of_starting_points)
+            ->default_value("closest"),
+        "selection stragey to select starting points (defaults to closest");
```

Comment:
Can we add a description of these strategy options in the readme files in the workflows directory?

Reply:
done